CIRG-PEG.json

```diff
@@ -0,0 +1,290 @@
+            "system": "http://loinc.org",
+            "code": "LA10140-4",
+            "display": "8"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA10141-2",
+            "display": "9"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA13942-0",
+            "display": "10"
+          }
+        }
+      ]
+    },
+    {
+      "linkId": "91146-1",
+      "code": [
+        {
+          "system": "http://loinc.org",
+          "code": "91146-1",
+          "display": "What number best describes how, during the past week, pain has interfered with your general activity?"
+        }
+      ],
+      "text": "What number best describes how, during the past week, pain has interfered with your general activity?",
+      "type": "choice",
+      "answerOption": [
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA6111-4",
+            "display": "0"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA6112-2",
+            "display": "1"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA6113-0",
+            "display": "2"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA6114-8",
+            "display": "3"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA6115-5",
+            "display": "4"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA10137-0",
+            "display": "5"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA10138-8",
+            "display": "6"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA10139-6",
+            "display": "7"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA10140-4",
+            "display": "8"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA10141-2",
+            "display": "9"
+          }
+        },
+        {
+          "valueCoding": {
+            "system": "http://loinc.org",
+            "code": "LA13942-0",
+            "display": "10"
+          }
+        }
+      ]
+    },
+    {
+      "linkId": "91147-9",
+      "code": [
+        {
+          "system": "http://loinc.org",
+          "code": "91147-9",
+          "display": "Mean score [PEG]"
```

Comment:
TODO for me: add another item for "Summed score" (not standard PEG), as that's what we implemented in PT per request by the PT clinical team.